pl_examples/basic_examples/mnist_datamodule.py

```diff
@@ -55,6 +55,9 @@ def __init__(
             normalize: If true applies image normalize
         """
         super().__init__(*args, **kwargs)
+        if platform.system() == "Windows":
+            # see: https://stackoverflow.com/a/59680818/4521646
+            num_workers = 0
```

Comment:
Not sure, but I thought this would be supported now since Microsoft take over PT support for windows?

Reply:
it started to break everything yesterday...